sdk/agrifood/arm-agrifood/CHANGELOG.md

```diff
@@ -11,22 +11,22 @@
 ### Other Changes
 
 ## 1.0.0-beta.4 (2022-11-14)
```

Comment:
Can you please add a description of your changes?

Reply:
These are just lint fixes by pre-release script, not changing anything as such.